src/fetchData.js

```diff
@@ -0,0 +1,22 @@
+import { display } from './domStuff';
```

Comment:
We are sort of doing some circular reference here. We don't normally wanna do that. By circular I mean in this file we are importing something from `domStuff` and in the `domstuff.js` file we are importing something from `fetchData.js`.  

Please see my below comment on `basic` function for more on this.